.circleci/config.yml

```diff
@@ -125,6 +133,7 @@ jobs:
     working_directory: ~/jest
     docker:
       - image: circleci/node:8
+    resource_class: large
```

Comment:
We can bump our test workers from `-w=2` to `-w=4` in package.json

Reply:
That in particular is just website build, isn't it? Might be worth doing for the test runs of course